src/com/tony/server/response/LogResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.tony.server.response;
+
+import com.tony.server.Logger;
+
+import java.util.HashMap;
+
+public class LogResponse extends Response{
+    private final Logger logger;
+
+    public byte[] respond(){
+        setBody(logger.getLog());
+        return ResponseBuilder.buildResponse(getStatusLine(),
+                getHeaders(), getBody()).getBytes();
+    } 
+    
+    public LogResponse(Logger logger) {
+        this.logger = logger;
+        setStatusLine("HTTP/1.1 200 OK");
```

Comment:
In most of your other Responses, you include a newline when setting your status line, but thats missing from this one. Also, if you look at the [HTTP spec](http://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html) for the response status line, it says that it should be followed by CRLF, which is carriage return and line feed or `\r\n`.